src/data/builder-tools.js

```diff
@@ -762,6 +762,14 @@ export const Showcases = [
     getstarted: null,
     tags: ["operatortool", "cli"],
   },
+  {
+    title: "Mesh",
+    description: "An Open-Source library by Martify Labs with the purpose of making building dApps on Cardano easier than ever. The Mesh library is engineered to simplify the development and implementation of blockchain technology, so you can focus on your project.",
+    preview: require("./builder-tools/mesh.jpeg"),
+    website: "https://mesh.martify.io/",
+    getstarted: "https://mesh.martify.io/guides",
```

Comment:
```suggestion
    getstarted: null,
```

There is a specific section in get started on developer portal dedicated to builder tools get started tutorials. The format is in markdown. Feel free to add your guide there as well, otherwise please write null in that field 😊

Reply:
A get started page on the developer portal is a work in progress. Will update this field when it is done. For now, setting `null` for `getstarted`.